superset-frontend/src/SqlLab/main.less

```diff
@@ -421,6 +421,7 @@ div.tablePopover {
       text-overflow: ellipsis;
       font-size: 16px;
       flex: 1;
+      width: 50px;
```

Comment:
May I suggest adding `width: 100%;` to the `.header-container` class instead? It seems more clear because the width of the table name will not be `50px` in practice.